spec/reactors/mempool/reactor.md

```diff
@@ -13,6 +13,11 @@ for details.
 Sending incorrectly encoded data or data exceeding `maxMsgSize` will result
 in stopping the peer.
 
+`maxMsgSize` equals `MaxTxBytes` (1MB) * `TxBatchSize` (100) + 4 (proto
```

Comment:
Can we set maxMsgSize as a config parameter independently from MaxTxBytes and TxBatchSize?

e.g. suppose I expect all txs to be 1KB, want a batch size of 1000, and want to allow occasional txs at the maxTxBytes (1MB).

In this setting, it seems reasonable to me to habe a maxMsgSize of around 1 to 2 MB, whereas this parameterization would net me 1GB.

Reply:
sorry, I just updated the spec to align with latest implementation. Now, there's a single parameter for maxMsgSize - MaxBatchSize (10MB).